test/images/jessie-dnsutils/VERSION

```diff
@@ -1 +1 @@
-1.6
+1.7
```

Comment:
why did we bump the version here? I don't see any changes to this image

Reply:
Thanks for pointing it out. Just reverted.